Packages/com.unity.multiplayer.samples.coop/Utilities/AdditiveSceneLoader.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+/// <summary>
+/// This NetworkBehavior, when added to a gameobject containing a collider (or multiple colliders) with the IsTrigger
```

Comment:
Let's note how this only happens on the server.

Also: gameobject-> GameObject

Reply:
There's still a mention of a "players". We can keep it generic for any interest colliders that enter the scene colliders.